static/src/javascripts/projects/common/views/acquisitions-epic-buttons.html

```diff
@@ -0,0 +1,17 @@
+<div class="contributions__amount-field">
+  <div>
+    <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member contributions__contribute--epic-member-top"
+      href="<%=membershipUrl%>"
```

Comment:
i am pretty sure `membershipUrl` won't be defined after the change to the buttons template